dbt/include/spark/macros/adapters.sql

```diff
@@ -229,9 +229,38 @@
   {% endfor %}
 {% endmacro %}
 
+{% macro get_column_comment_sql(column_name, column_dict) -%}
+  {% if (column_name|upper in column_dict) -%}
+    {% set matched_column = column_name|upper -%}
+  {% elif (column_name|lower in column_dict) -%}
+    {% set matched_column = column_name|lower -%}
+  {% elif (column_name in column_dict) -%}
+    {% set matched_column = column_name -%}
+  {% else -%}
+    {% set matched_column = None -%}
+  {% endif -%}
+  {% if matched_column and column_dict[matched_column]["description"] -%}
```

Comment:
We should be cautious about `"description"` being a key here, just like we're cautious about `matched_column` being a key. Instead of using the index, can we do `.get("description")`?

Reply:
We check that `matched_column` exists as a key in `column_dict`. But we don't check that `"description"` is a key in `column_dict[matched_column]`. The latter would throw a key error if `"description"` is not a key.

Reply:
I'd tested it, and in this case the condition is just falsy and column_comment_clause stays unset.